app/Imports/ChildrenImport.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Imports;
+
+use App\Models\Child;
+use Illuminate\Support\Collection;
+use Maatwebsite\Excel\Concerns\ToCollection;
+use \PhpOffice\PhpSpreadsheet\Shared\Date;
+
+class ChildrenImport implements ToCollection
+{
+    public function collection(Collection $rows)
+    {
+        $listChild = $rows->splice('8');
+        dd($rows->toArray());
+        foreach ($listChild as $row)
+        {
+            Child::create([
+                'number' => $row['0'],
```

Comment:
không nên dùng số làm key, vì sau này nếu có muốn đổi thứ tự cột thì sẽ phải sửa lại toàn bộ => nên đặt key là text, đặt tên mỗi cột 1 text. VD: 'name', 'holy_name';

Reply:
Vâng anh, em cảm ơn ạ